### DISCUSSION TITLE
Use of webview in winform c++ /clr

### DISCUSSION THREAD

**Discussion by POWER-KI** [MicrosoftEdge/WebView2Feedback#3857] [Q&A]
There is a way to use webview in a winform c++ CLI /clr app? 
My problem is that WRL, which complains it can't be compiled with /clr,
Someone has find a solutions?
Thank you

**Comment by victorhuangwq** [comment]
@johna-ms could you take a look at this?

**Comment by johna-ms** [comment]
Hi POWER-KI, we haven't tried to run WebView2 in this framework. If it is incompatible with WRL, you probably can't use WebView2 with this.
